shell/components/ExplorerMembers.vue

```diff
@@ -70,13 +66,15 @@ export default {
           cluster: this.$store.getters['currentCluster'].id
         }
       },
-      resource:                    MANAGEMENT.CLUSTER_ROLE_TEMPLATE_BINDING,
-      clusterRoleTemplateBindings: [],
+      resource: MANAGEMENT.CLUSTER_ROLE_TEMPLATE_BINDING,
       VIRTUAL_TYPES
     };
   },
 
   computed: {
+    clusterRoleTemplateBindings() {
+      return this.$store.getters[`rancher/all`](NORMAN.CLUSTER_ROLE_TEMPLATE_BINDING).map(b => b.clusterroletemplatebinding) ;
+    },
     ...mapGetters(['currentCluster']),
     filteredClusterRoleTemplateBindings() {
       return this.clusterRoleTemplateBindings.filter(
```

Comment:
When I add a user the dash breaks in both windows with the error message: `Cannot read properties of undefined (reading 'clusterName')`

Changing to `b?.clusterName` appears to be enough to resolve it then everything works well (deleting members is fine too). If I had to guess, I think there is probably a split second after saving where the dashboard has a norman cluster role template binding, but it hasn't gotten the binding from steve yet.